openapi-v2.yaml

```diff
@@ -1983,6 +1983,79 @@ definitions:
         x-omitempty: false
         $ref: "#/definitions/PaginationMetadata"
 
+  GroupActivityLogResponse:
+    description: Object containing activity logs of a group along with the pagination metadata
+    type: object
+    properties:
+      activitylogs:
+        description: Array of GroupActivityLog
+        type: array
+        x-omitempty: false
+        items:
+          $ref: "#/definitions/GroupActivityLog"
+      pagination_metadata:
+        x-omitempty: false
```

Comment:
@alekshs this should be false or true?

Reply:
There should always be a metadata object i guess